<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ИВТ-41 Ассистент</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../pics/binbetbot.jpg</normaloff>../../pics/binbetbot.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QListWidget" name="chatWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Бот говорит вслух</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leMessage"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pbMicro">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../pics/Microphone-icon.png</normaloff>../../pics/Microphone-icon.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbText">
          <property name="text">
           <string>Отправить текст</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="startTele"/>
   <addaction name="DB"/>
   <addaction name="settings"/>
   <addaction name="action_SQL"/>
   <addaction name="supportForm"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="dbRefresh"/>
  </widget>
  <action name="startTele">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../pics/starttele.png</normaloff>../../pics/starttele.png</iconset>
   </property>
   <property name="text">
    <string>Запуск бота в Telegarm</string>
   </property>
  </action>
  <action name="DB">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/Databaseicon.png</normaloff>../../pics/Databaseicon.png</iconset>
   </property>
   <property name="text">
    <string>База данных</string>
   </property>
   <property name="toolTip">
    <string>База данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="settings">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/SettingDatabaseicon.png</normaloff>../../pics/SettingDatabaseicon.png</iconset>
   </property>
   <property name="text">
    <string>Натройка подключения к MySQL</string>
   </property>
   <property name="toolTip">
    <string>Натройка подключения к MySQL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="supportForm">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/support.png</normaloff>../../pics/support.png</iconset>
   </property>
   <property name="text">
    <string>Справка о программе</string>
   </property>
   <property name="toolTip">
    <string>Справка о программе</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_SQL">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/sql.png</normaloff>../../pics/sql.png</iconset>
   </property>
   <property name="text">
    <string>Ввести SQL запрос вручную</string>
   </property>
   <property name="toolTip">
    <string>Ввести SQL запрос вручную</string>
   </property>
  </action>
  <action name="sendAction">
   <property name="text">
    <string>Отправить сообщение</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="dbRefresh">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/dbrefresh.png</normaloff>../../pics/dbrefresh.png</iconset>
   </property>
   <property name="text">
    <string>Переподключиться к БД</string>
   </property>
   <property name="toolTip">
    <string>Переподключить бота к БД</string>
   </property>
  </action>
  <action name="stopTele">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../pics/stoptele.png</normaloff>../../pics/stoptele.png</iconset>
   </property>
   <property name="text">
    <string>Остановить бота</string>
   </property>
   <property name="toolTip">
    <string>Остановить работу бота в телеграмме</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
